docs/tutorial/tutorial-1.rst

```diff
@@ -0,0 +1,104 @@
+.. _tutorial-1:
+
+=================
+Your first bridge
+=================
+
+In this example, we're going to use Rubicon to access the Objective-C
+Foundation library, and the `NSURL` class in that library. `NSURL` is the
+class used to represent and manipulate URLs.
+
+This tutorial assumes you've set up your environment as described in the
+:ref:`Getting started guide <get-started>`.
+
+Accessing NSURL
+---------------
+
+Start Python, and use ctypes to import a framework into the Python
+interpreter:
+
+.. code-block:: python
+
+    >>> from ctypes import cdll, util
+    >>> cdll.LoadLibrary(util.find_library("Foundation"))
```

Comment:
This isn't necessary anymore, Foundation is already loaded in `rubicon.objc.runtime`.

Reply:
ack